### ISSUE TITLE
[V5.6] Export Profile notification emails are not being sent

### ISSUE THREAD

**Issue by icreatestuff** [solspace/craft3-freeform#1530]
### What happened?

We have an export profile notification created to send the report on a daily basis as a CSV to multiple emails (separated to new lines within the 'Emails to notify' field).

None of the email addresses listed receive the notification. 

Other emails within Craft are sent without issue so we don't think this is a problem with the configuration of emails within Craft's settings or the email service provider.

**Diagnostics**
![Screenshot 2024-09-25 at 13 35 20](https://github.com/user-attachments/assets/b760ebef-c92b-4d32-8d14-84d82fdf73ee)
### 

### Errors and Stack Trace (if available)

```shell
2024-09-25 02:21:12 [web.ERROR] [TypeError] Cannot assign null to property Solspace\Freeform\Library\DataObjects\NotificationTemplate::$uid of type string {
    "trace":[
        "#0 /srv/app/htdocs/vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Notifications/Export/ExportNotifications.php(86): Solspace\\Freeform\\Library\\DataObjects\\NotificationTemplate::fromRecord(Object(Solspace\\Freeform\\Records\\NotificationTemplateRecord))",
        "#1 [internal function]: Solspace\\Freeform\\Bundles\\Notifications\\Export\\ExportNotifications->handleNotifications(Object(yii\\base\\Event))",
        "#2 /srv/app/htdocs/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Array, Object(yii\\base\\Event))",
        "#3 /srv/app/htdocs/vendor/yiisoft/yii2/base/Component.php(650): yii\\base\\Event::trigger('craft\\\\web\\\\Appli...', 'afterRequest', Object(yii\\base\\Event))",
        "#4 /srv/app/htdocs/vendor/craftcms/cms/src/base/ApplicationTrait.php(544): yii\\base\\Component->trigger('afterRequest', Object(yii\\base\\Event))",
        "#5 /srv/app/htdocs/vendor/yiisoft/yii2/base/Application.php(387): craft\\web\\Application->trigger('afterRequest')",
        "#6 /srv/app/htdocs/web/index.php(12): yii\\base\\Application->run()",
        "#7 {main}"
    ],
    "memory":8514872,
    "exception":"[object] (TypeError(code: 0): Cannot assign null to property Solspace\\Freeform\\Library\\DataObjects\\NotificationTemplate::$uid of type string at /srv/app/htdocs/vendor/solspace/craft-freeform/packages/plugin/src/Library/DataObjects/NotificationTemplate.php:54)"
}
```

### How can we reproduce this?

1. Create an export profile for any existing form
2. Create a daily export notification for that profile set to CSV type

### Freeform Edition

Pro

### Freeform Version

5.6.1

### Craft Version

4.12.3

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4.1.22

**Comment by kjmartens** [comment]
Thanks for reporting this @icreatestuff.

We are checking into it now. 🙂

**Comment by kjmartens** [comment]
This is now resolved in Freeform 5.6.4+ 🙂

**Comment by icreatestuff** [comment]
Brilliant, thank you!
